packages/design-system/src/components/N8nButton/Button.stories.ts

```diff
@@ -0,0 +1,197 @@
+/* tslint:disable:variable-name */
+import N8nButton from './Button.vue';
+import { action } from '@storybook/addon-actions';
+import { StoryFn } from "@storybook/vue";
+
+export default {
+	title: 'Atoms/Button',
+	component: N8nButton,
+	argTypes: {
+		label: {
+			control: 'text',
+		},
+		title: {
+			control: 'text',
+		},
+		type: {
+			control: 'select',
+			options: ['primary', 'secondary', 'tertiary', 'success', 'warning', 'danger'],
+		},
+		size: {
+			control: {
+				type: 'select',
+				options: ['mini', 'small', 'medium', 'large', 'xlarge'],
+			},
+		},
+		outline: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		text: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		loading: {
+			control: {
+				type: 'boolean',
+			},
+		},
+		icon: {
+			control: {
+				type: 'text',
+			},
+		},
+		fullWidth: {
+			type: 'boolean',
```

Comment:
can be removed or replaced with block

Reply:
in fact, you can probably delete most of the props here... storybook automatically adds props that are specified.. (it did not do that when I wrote this component initially)

Reply:
Good point! Removed props that are auto-generated.